Sources/hostmgr/HostMgrCommand.swift

```diff
@@ -52,7 +52,7 @@ struct InitCommand: ParsableCommand {
     )
 
     func run() throws {
-        if StateManager.configurationFileExists {
+        if ConfigurationRepository.configurationFileExists {
```

Comment:
Would it be appropriate to remove `configurationFileExists` from `StateManager` in the context of this PR?

https://github.com/Automattic/hostmgr/blob/4bc3e42b494d8a867fd13c7e6438e04c7e69c9bc/Sources/libhostmgr/StateManager.swift#L18-L20

If so, it would also be necessary to update `HostMgrCommand` to use `ConfigurationRepository`, which is neat, I think because it would also give us a chance to remove the access to `Configuration.shared` that it makes later on in favor of the new `try ConfigurationRepository.getConfiguration()`

https://github.com/Automattic/hostmgr/blob/d93260dda75d92f634f2a61a94a290da97c41039/Sources/hostmgr/HostMgrCommand.swift#L55-L63

Reply:
[It ended up] in(https://github.com/Automattic/hostmgr/pull/50/files#diff-9d2950e69cad848b8f77c03632b470a59040fed22beb5065081f0548a10fee91L55) #50, I _think_ because that's where I was removing references to `StateManager`? 

If it's ok with you, I think I'll land this one and get that one open for review right away, which should resolve the oddity